lib/index.js

```diff
@@ -1 +1,37 @@
 var $ = require ('jquery');
+
+const canvas = document.getElementById('arkannoyed-canvas');
+const context = canvas.getContext('2d');
+
+const World = require('./world');
+const Ball = require('./ball');
+const Paddle = require('./paddle');
+
+var world = new World(canvas.width, canvas.height);
+// var ball = world.ball;
+
+function genesis(context, world) {
+  context.clearRect(0, 0, world.width, world.height);
+  context.fillStyle = 'blanchedalmond';
+  context.fillRect(0, 0, world.width, world.height);
+  function drawPaddle(){
+    world.paddle.x = world.width/2 - world.paddle.width/2;
+    world.paddle.y = world.height - world.paddle.height*2;
+    context.fillStyle = 'blue';
+    context.fillRect(world.paddle.x, world.paddle.y, world.paddle.width, world.paddle.height);
+  }
+  function drawBall(){
+    world.ball.x = world.paddle.x + world.paddle.width/2 - world.ball.width/2;
```

Comment:
the ball's defaults should be locked to the paddle already without needing this work re-done here.